src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -289,16 +289,13 @@ export async function promptForArguments( preselectedOptions: InstanceOptions, d
 	}
 
 	let multisiteText = 'Multisite';
-	let multisiteDefault = DEV_ENVIRONMENT_DEFAULTS.multisite;
-
 	if ( defaultOptions.title ) {
 		multisiteText += ` (${ defaultOptions.title } ${ defaultOptions.multisite ? 'IS' : 'is NOT' } multisite)`;
-		multisiteDefault = defaultOptions.multisite;
 	}
 
 	const instanceData: InstanceData = {
 		wpTitle: preselectedOptions.title || await promptForText( 'WordPress site title', defaultOptions.title || DEV_ENVIRONMENT_DEFAULTS.title ),
-		multisite: preselectedOptions.multisite !== undefined ? preselectedOptions.multisite : await promptForBoolean( multisiteText, !! multisiteDefault ),
+		multisite: resolveMultisite( preselectedOptions.multisite ?? await promptForTextBoolean( multisiteText, defaultOptions.multisite || DEV_ENVIRONMENT_DEFAULTS.multisite ) ),
```

Comment:
We currently don't seem to be doing any validation of the imput from user.

I tested it and it takes in the `banana` for multisite and turns it into `false`.

I think what we should do instead is go through similar re-prompting as we do if you provide an incorrect path to client-code or mu-plugins.

Also there we can list all the valid options properly without making the first prompt heavy to read.

![image](https://user-images.githubusercontent.com/19240162/232383392-b95fc4ca-7b36-4ed7-9f19-e03916852d24.png)